camayoc/qcs_models.py

```diff
@@ -219,12 +226,26 @@ def equivalent(self, other):
             )
 
         password_matcher = re.compile(MASKED_PASSWORD_OUTPUT)
-        for key, value in self.fields().items():
-            if key == 'password' and other.get(key) is not None:
+        local_items = self.fields()
+        local_keys = local_items.keys()
+        other_keys = other.keys()
+        all_keys = set().union(*[local_keys, other_keys])
```

Comment:
You could accomplish the same with the code below:

```python
In [1]: l1 = [1, 2, 3]

In [2]: l2 = [2, 3, 4]

In [3]: set(l1).union(l2)
Out[3]: {1, 2, 3, 4}
```

Reply:
hmm let me try that out. so you are saying `set(local_keys).union(other_keys)`

Reply:
yes